sdk/storage/azure-storage-blob/azure/storage/blob/_shared/utils.py

```diff
@@ -0,0 +1,608 @@
+# -------------------------------------------------------------------------
```

Comment:
This file has been refactored away. Did you make an edits to it? Or is it just here as merge artifact?